spec/tbd_tests_spec.rb

```diff
@@ -4708,6 +4601,90 @@
     expect(vals[:sillconvex]).to  be_within(0.001).of(0.391)     # :fenestration
   end
 
+  it "can flag polygon overlaps" do
+    # Testing  'fits?' & 'overlaps?' functions (framedivider.rb).
+    model = OpenStudio::Model::Model.new
+
+    # 10m x 10m parent vertical (wall) surface
+    vec = OpenStudio::Point3dVector.new
+    vec << OpenStudio::Point3d.new(  0,  0, 10)
+    vec << OpenStudio::Point3d.new(  0,  0,  0)
+    vec << OpenStudio::Point3d.new( 10,  0,  0)
+    vec << OpenStudio::Point3d.new( 10,  0,  10)
+    wall = OpenStudio::Model::Surface.new(vec, model)
+
+    # 1m x 2m corner door
+    vec = OpenStudio::Point3dVector.new
+    vec << OpenStudio::Point3d.new(  0,  0,  2)
+    vec << OpenStudio::Point3d.new(  0,  0,  0)
+    vec << OpenStudio::Point3d.new(  1,  0,  0)
+    vec << OpenStudio::Point3d.new(  1,  0,  2)
+    door1 = OpenStudio::Model::SubSurface.new(vec, model)
+
+    # Door1 fits?, overlaps?
+    TBD.clean!
+    expect(fits?(door1.vertices, wall.vertices)).to be(true)
+    expect(overlaps?(door1.vertices, wall.vertices)).to be(true)
+    expect(TBD.status).to eq(0)
+
+    # Order of arguments doesn't matter.
```

Comment:
I think you want the order to matter so you can test the sub surface is inside the surface but not vice versa